src/ds5/ds5-auto-calibration.cpp

```diff
@@ -1677,7 +1786,14 @@ namespace librealsense
                 uint8_t* table = (uint8_t*)(calibration.data() + sizeof(table_header));
                 command write_calib(ds::CALIBRECALC, 0, 0, 0, 0xcafecafe);
                 write_calib.data.insert(write_calib.data.end(), (uint8_t*)table, ((uint8_t*)table) + hd->table_size);
-                _hw_monitor->send(write_calib);
+                try
+                {
+                    _hw_monitor->send(write_calib);
+                }
+                catch(...)
+                {
+                    LOG_ERROR("Flashing coefficients_table_id failed");
```

Comment:
Does this happen? Do we know why?

Reply:
It is often DEVICE_BUSY error, probably due to priority management